### PR TITLE
Remove tech preview for Azure KMS

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
https://issues.redhat.com/browse/RHSTOR-5938

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2024-05-31 04-24-42

**Comment by nthomas-redhat** [comment]
@TimothyAsirJeyasing , you have to account for this ( https://github.com/red-hat-storage/odf-console/pull/1420 )  as well right?

File: packages/odf/components/kms-config/kms-config.tsx [hunk 1]
```diff
@@ -1,12 +1,9 @@
 import * as React from 'react';
-import { TechPreviewBadge } from '@odf/shared';
 import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
 import {
   FormGroup,
-  MenuToggle,
-  MenuToggleElement,
-  Select,
```
[the remark this answers is not in the document]
Comment (TimothyAsirJeyasing) [reply packages/odf/components/kms-config/kms-config.tsx]: Yes its good to keep it. I will add this back

**Comment by TimothyAsirJeyasing** [comment]
image

**Comment by TimothyAsirJeyasing** [comment]
> @TimothyAsirJeyasing , you have to account for this ( #1420 ) as well right?

yes, sure

**Comment by SanjalKatiyar** [comment]
add epic/story link to description...

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**